hack/apply-orchestration-workflows

```diff
@@ -17,6 +17,8 @@ function install_orchestration_workflows () {
     --set "env=${env}" \
     --set "clinvar.enable=true" \
     --set 'clinvar.schedule="0 8 * * *"'
+    --set "encode.enable=true" \
+    --set 'encode.schedule="0 8 * * *"'
```

Comment:
Should these values be the same for encode that they are for clinvar?

Reply:
Seems fine to me!